controllers/daemonset.go

```diff
@@ -7,7 +7,7 @@ import (
 
 	"github.com/go-openapi/swag"
 	ootov1alpha1 "github.com/qbarrand/oot-operator/api/v1alpha1"
-	"github.com/qbarrand/oot-operator/controllers/constants"
```

Comment:
The controller/daemonset pkg isn't a controller but rather an internal package. Shouldn't it be moved to internal/ as well?

Reply:
i thought about that , but could not find where to move it:  internal/daemonset?

Reply:
> internal/daemonset

Yes, why not (how does it differ from other packages moved to `internal/`)?

Reply:
fixed